docs/designers-developers/developers/tutorials/create-block/readme.md

```diff
@@ -14,7 +14,7 @@ From your plugins directory, to create your block run:
 npx @wordpress/create-block starter-block
 ```
 
-The above command will ask you a few questions to customize, then will create a new directory called `starter-block`, installs the necessary files, and builds the block plugin.
+The above command creates a new directory called `starter-block`, installs the necessary files, and builds the block plugin. If you run the command without a name, it will trigger an interactive mode to prompt you.
```

Comment:
It might be a more detailed message about what "a name" is. How about it gets replaced with `starter-block`?

Reply:
Updated. 👍